config/deploy.rb

```diff
@@ -44,6 +44,9 @@
   'config/property_permissions.yml',
   'config/redis.yml',
   'config/resque-pool.yml',
+  'config/resque-web.rb',
+  'config/initializers/resque_config.rb',
+  'config/initializers/resque_admin.rb',
```

Comment:
Well, because these files aren't going to change that much, and ought to be preserved through deploys of app code changes.